device/esp_tinyusb/test_app/main/test_tud_cb.c

```diff
@@ -0,0 +1,28 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Espressif Systems (Shanghai) CO LTD
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stdio.h>
+#include <string.h>
+#include "tinyusb.h"
+#include "tusb_tasks.h"
+#include "test_bvalid_sig.h"
+#include "test_descriptors_config.h"
+
+// Invoked when device is mounted
+void tud_mount_cb(void)
```

Comment:
@tore-espressif  I have added the test to verify all possible configuration options. 
The base flag of successful mounting - `tud_mount_cb`. 
But I need that callback in both tests (`bvalid_signal` and `descriptors_config`), so I decided to make the general callback and then to call callbacks within the specific test from it. 

I am not 100% sure that this is the best way, maybe you can suggest something better.

Reply:
Now the 2 tests are interfering with one another: running the 'descriptor' test case will modify value of `dev_mounted/umounted` inside the 'bvalid' testcases.

If you plan to merge it like this, please track the follow-up somewhere, so we can fix soon